Modules/faulthandler.c

```diff
@@ -1334,22 +1334,32 @@ PyInit_faulthandler(void)
 #ifdef MS_WINDOWS
     /* RaiseException() codes (prefixed by an underscore) */
     if (PyModule_AddIntConstant(m, "_EXCEPTION_ACCESS_VIOLATION",
-                                EXCEPTION_ACCESS_VIOLATION))
+                                EXCEPTION_ACCESS_VIOLATION)) {
+        Py_DECREF(m);
         return NULL;
```

Comment:
Hum, what do you think of adding a "error:" label which does the DECREF + return NULL?

Reply:
I actually prefer that. I just wanted to keep the diffs small for all of these.

Should I make the change?

Reply:
Done.